src/Platform/Microsoft.Testing.Platform/OutputDevice/OutputDeviceManager.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.Testing.Platform.OutputDevice;
 
-internal sealed class PlatformOutputDeviceManager : IPlatformOutputDeviceManager
+internal sealed class PlatformOutputDeviceManager
```

Comment:
It was there because of potential future plans to make `IPlatformOutputDeviceManager` public and allow framework authors to plug their own output device through `SetPlatformOutputDevice`.

@Evangelink @MarcoRossignoli Are you fine dropping the interface for now, and re-introduce later if we want to expose it?

Reply:
@Evangelink @MarcoRossignoli bump

Reply:
Given shift in priorities, it's probably safe to remove for now.